_layouts/news-item.html

```diff
@@ -0,0 +1,42 @@
+---
+layout: page
+---
+<article itemscope itemtype="http://schema.org/Article">
+  <h2 itemprop="headline">
```

Comment:
```suggestion
  <h2 itemprop="headline">
```

i believe this should likely be an h1